N1C WAF Security Dashboard Release (#1834)
nginx/documentation: content/nginx-one-console/waf-integration/waf-security-dashboard/find-event-by-support-id.md

If no event is found, the most likely causes are that the Support ID does not belong to a WAF-inspected request, the event is older than the 90-day retention window, or the Support ID was mistyped.

---

## What the detail panel shows

The Security Event detail panel surfaces every field stored on the event:

- **Request** — request method, URI, host, headers, and the **Raw Request** payload as captured by F5 WAF for NGINX. The raw request is the same payload that would appear in a NAP security log on the data plane.
- **Source** — client IP, X-Forwarded-For chain, and a geolocation map showing the country derived from the client IP.
- **Time of Request** — when F5 WAF for NGINX produced the event.
- **Triggered violations and signatures** — every violation and signature that fired on the event, with their full attributes (name, accuracy, risk, CVE, context).
- **Threat campaigns** — any threat campaigns matched by the event.
- **Policy and outcome** — the F5 WAF for NGINX policy that produced the event, the request status (`blocked` or `alerted`), and the outcome reason.

---

> ## Where to find a Support ID

Reviewer: Support Id can also be found in the security dashboard event logs, if we want to mention that

Reviewer 2: added

---
nd-content-type: how-to
nd-docs: DOCS-000
nd-product: NONECO
title: Find a security event by Support ID
description: "Look up an F5 WAF for NGINX security event in NGINX One Console using its Support ID."
weight: 500
toc: true

> nd-keywords: "support ID, security event, F5 WAF for NGINX, security dashboard, troubleshooting, event lookup"

Reviewer 3: ```suggestion
nd-keywords: "support ID, security event, F5 WAF for NGINX, security dashboard, event lookup, Support ID Details, getSecurityEvent, listSecurityEvents, WAF block, rejection page, triage, 90-day retention"
```

Reviewer 2: accepted your suggestion.

nd-summary: >
  Use the Support ID Details page in NGINX One Console to look up a single F5 WAF for NGINX security event by its Support ID.
  This is the fastest way to drill from a customer report, an upstream log line, or an alert into the full WAF event record.
  This guide covers what a Support ID is, where to find one, and how to use the lookup page.
nd-audience: operator
---

## Overview

> Every request that F5 WAF for NGINX inspects is assigned a unique **Support ID**. The Support ID stays with the request through every system that touches it — F5 WAF for NGINX security logs, NGINX access logs (when configured), upstream application logs, and the security monitoring dashboard. NGINX One Console exposes a dedicated **Support ID Details** page so you can paste a Support ID and immediately see the full WAF event record without filtering through the dashboard.

Reviewer 3: Use the **Support ID Details** page in NGINX One Console to look up a single F5 WAF for NGINX security event by its **Support ID**. F5 WAF for NGINX assigns a unique **Support ID** to every inspected request, and that ID travels through every system that touches the request: security logs, NGINX access logs, upstream application logs, and the security monitoring dashboard.

Reviewer 2: accepted

Use this page when you already know the ID of the event you want to inspect — for example, from a customer support case, a raw F5 WAF for NGINX log line, an alert payload, or an upstream application that captured the ID from a request header. To explore events without a known ID, use the Event Logs tab on the [security monitoring dashboard]({{< ref "/nginx-one-console/waf-integration/waf-security-dashboard/dashboard-metrics-reference.md#event-logs-tab" >}}) instead.

---

## Before you begin

Before you begin, ensure you have:

- **A Support ID**: A numeric Support ID for the event you want to inspect. See [Where to find a Support ID](#where-to-find-a-support-id) below.
- **Security monitoring set up**: F5 WAF for NGINX security events must already be flowing into NGINX One Console for the event to be available. See [Set up security monitoring]({{< ref "/nginx-one-console/waf-integration/waf-security-dashboard/set-up-security-monitoring.md" >}}).

> - **Within the retention window**: Security events are retained for 90 days. Events older than 90 days are no longer available.

Reviewer 3: This is not a requirement, is more like a callout. Maybe move it to step 3?

Reviewer 2: yup

---

## Look up an event by Support ID

1. In NGINX One Console, go to **WAF** > **Support ID Details**.
2. Paste the Support ID into the **Enter a Support ID** field.
3. The page displays the **Security Event** detail panel for the matching event, including the request, source, and raw request data.

> If no event is found, the most likely causes are that the Support ID does not belong to a WAF-inspected request, the event is older than the 90-day retention window, or the Support ID was mistyped.

Reviewer 3: ```suggestion
### Troubleshooting

If no event is found, the most likely causes are:

- The Support ID does not belong to a WAF-inspected request
- The event is older than the 90-day retention window
- The Support ID was mistyped
```

Reviewer 2: accepted

---

## What the detail panel shows

> 

Reviewer 3: `### What the detail panel shows`
`### Where to find a support ID`
`### Use cases`

These 3 sections are reference/concept. Not sure if they should be in a how-to guide or moved to one or several separate docs

cc @travisamartin

Reviewer 2: will need to revisit this.
